strawberryfields/backends/tfbackend/circuit.py

```diff
@@ -577,7 +577,7 @@ def measure_fock(self, modes, select=None, **kwargs):
 
         if select is not None:
             # just use the supplied measurement results
-            meas_result = select
+            meas_result = tf.convert_to_tensor([[i] for i in select], dtype=tf.int64)
```

Comment:
I've seen `ops.def_type` being used later, just curious if it would be good to have it here too?